package.json

```diff
@@ -101,7 +101,7 @@
     "test-integration-with-reports": "npm run test-integration -- --reporter mocha-junit-reporter --reporter-options mochaFile=reports/integration-results.xml",
     "test-e2e-with-reports": "npm run test-e2e -- --reporter mocha-junit-reporter --reporter-options mochaFile=reports/e2e-results.xml",
     "test-unit": "mocha test/unit/**/*.test.ts",
-    "test-unit-watch": "npm run test-unit -- -w --watch-files '**/*.ts'",
+    "test-unit-watch": "npm run test-unit -- -w --watch-files '**/*.ts'  --forbid-only false",
```

Comment:
How do we do this in other repos? I am not sure if I have seen this flag before 🤔

Reply:
allowing `.only` in tests.

Reply:
This flag allows you to use .only, by default should be false in mocha if I am not mistaken, but in oclif is not, and it was quite painful for local development

Reply:
I'm against adding this under version control 😁 
What about calling the script locally with 

```bash
npm run test-unit-watch -- --test-unit-watch false
```

instead :)

Reply:
I will remove it, this is not related to the introduced change anyway 👍